presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -264,6 +264,22 @@ public void testSimpleCoral()
                 queryAssert -> queryAssert.containsOnly(row("hive")));
     }
 
+    @Test
```

Comment:
nit: Specify `HIVE_VIEWS` group.
```suggestion
    @Test(groups = HIVE_VIEWS)
```